libraries/lib-wave-track/WaveTrack.h

```diff
@@ -612,11 +613,14 @@ class WAVE_TRACK_API WaveTrack final : public WritableSampleTrack
    //! Whether all clips have a common rate
    bool RateConsistencyCheck() const;
 
+   void PushBack(WaveClipHolder clip);
+
    SampleBlockFactoryPtr mpFactory;
 
    wxCriticalSection mFlushCriticalSection;
    wxCriticalSection mAppendCriticalSection;
    double mLegacyProjectFileOffset;
+   std::optional<double> mProjectTempo;
```

Comment:
This duplicates the member in base class Track.  Perhaps define a const accessor function instead

Reply:
Good idea.